lib/instrumentation/index.spec.ts

```diff
@@ -74,9 +79,16 @@ describe('instrumentation/index', () => {
     const delegateProvider = proxyProvider.getDelegate();
     expect(delegateProvider).toBeInstanceOf(NodeTracerProvider);
     const nodeProvider = delegateProvider as NodeTracerProvider;
-    const provider = nodeProvider.getActiveSpanProcessor();
-    expect(provider).toBeInstanceOf(MultiSpanProcessor);
```

Comment:
why not leave this ?

Reply:
This has been a way to test implicitly the number of exporters, which is now not necessary anymore.